Sources/BrowserServicesKit/ContentBlocking/ContentBlockerRulesManager.swift

```diff
@@ -377,6 +388,9 @@ public class ContentBlockerRulesManager: CompiledRuleListsSource {
                                                                                                 unprotectedSitesHash: nil))
             }
 
+            // if task is main tds task, extract time and iteration count from the result and fire the pixel
+            // can result be a struct? can we add fields to compliatiotask instead?
+            // can we sen dpixels from compialtion task?
```

Comment:
I think this is not an intended change :)

Reply:
Done